eth-brownie/brownie
# Support multiple vyper versions

### What I did
Allow multiple vyper versions in a project, and support older versions (`>=0.1.0-beta.16`)

### How I did it
Use [`vvm`](https://github.com/vyperlang/vvm) instead of directly importing vyper.

### How to verify it
Run tests.  I added and updated many cases. 

Note that especially on Windows this is causing significant slowdown in the test suite.  In the near future I'll do a further refactor of the tests so that vyper related tests run on their own workers.
